velox/expression/tests/utils/ArgumentTypeFuzzer.h

```diff
@@ -69,6 +69,8 @@ class ArgumentTypeFuzzer {
   /// randomly generated type.
   void determineUnboundedTypeVariables();
 
+  void determineUnboundedIntegerVariables(const exec::TypeSignature& type);
```

Comment:
nit: add a small comment on how its different from the one above, i.e., only applies to Decimal types and only operates on the type passed to it.

Reply:
Added

```
  // Bind integer variables used in specified decimal(p,s) type signature to
  // randomly generated values if not already bound.
  // Noop if 'type' is not a decimal type signature.
  void determineUnboundedIntegerVariables(const exec::TypeSignature& type);
```